lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
 
     # Let the XQueue know that we handled the response
     return JsonResponse({'return_code': 0})
+
+
+# pylint: disable=too-many-statements, bad-continuation
+@login_required
+def render_html_view(request):
+    """
+    This view generates an HTML representation of the specified student's certificate
+    If a certificate is not available, we display a "Sorry!" screen instead
+    """
```

Comment:
Ok, so instead of importing the Django settings module and then doing a settings.FEATURES.get() check, we'd instead import the specific configuration model and check its 'enabled' property?